pkg/cli/pod.go

```diff
@@ -145,11 +150,12 @@ set -euo pipefail
 # Download all packages so we can avoid rebuilding them.
 mkdir -p ./packages/{{.arch}}
 gsutil -m rsync -r {{.bucket}}{{.arch}} ./packages/{{.arch}} || true
-gsutil cp {{.bucket}}{{.signingKeyName}}.rsa.pub .
+gsutil cp {{.publicKeyBucket}}{{.signingKeyName}}.rsa.pub .
```

Comment:
If you do --signing-key=foo/bar/key.rsa.pub does that work?

Reply:
Unfortunately not because the public key is in the folder above